File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/UserLoginResponseContent.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class UserLoginResponseContent extends BaseResponseContent {

    public static final String USER_SESSION = "user_session";
    public static final String USER_SUID = "user_suid";
    public static final String USER_FUNCTIONS = "user_functions";

    public static final String USER_PREFS = "user_prefs";
    public static final String USER_FULLNAME = "user_fullname";
    public static final String USER_NAME = "user_name";


    @SerializedName(USER_SESSION)
    private String user_session;

    @SerializedName(USER_SUID)
    private String user_suid;


    @SerializedName(USER_FUNCTIONS)
    private List<String> user_functions;


    @SerializedName(USER_PREFS)
    private UserPrefs mUser_prefs;

    @SerializedName(USER_FULLNAME)
    private String user_fullname;

    @SerializedName(USER_NAME)
    private String user_name;

    public String getUserSession() {
        return user_session;
    }

    public String getUserSuid() {
        return user_suid;
    }

    public List<String> getListUserFunctions() {
        return user_functions;
    }


    public UserPrefs getUserPrefs() {
        return mUser_prefs;
    }


    public String getUserFullname() {
        return user_fullname;
    }

    public String getUserName() {
        return user_name;
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/UserLoginResponseContent.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class UserLoginResponseContent : BaseResponseContent() {
    companion object {
        const val USER_SESSION = "user_session"
        const val USER_SUID = "user_suid"
        const val USER_FUNCTIONS = "user_functions"
        const val USER_PREFS = "user_prefs"
        const val USER_FULLNAME = "user_fullname"
        const val USER_NAME = "user_name"
    }

    @SerializedName(USER_SESSION)
    private val user_session: String? = null

    @SerializedName(USER_SUID)
    private val user_suid: String? = null

    @SerializedName(USER_FUNCTIONS)
    private val user_functions: List<String?>? = null

    @SerializedName(USER_PREFS)
    private val mUser_prefs: UserPrefs? = null

    @SerializedName(USER_FULLNAME)
    private val user_fullname: String? = null

    @SerializedName(USER_NAME)
    private val user_name: String? = null

    fun getUserSession(): String? {
        return user_session
    }

    fun getUserSuid(): String? {
        return user_suid
    }

    fun getListUserFunctions(): List<String?>? {
        return user_functions
    }

    fun getUserPrefs(): UserPrefs? {
        return mUser_prefs
    }

    fun getUserFullname(): String? {
        return user_fullname
    }

    fun getUserName(): String? {
        return user_name
    }
}